src/utils/object.utils.ts

```diff
@@ -120,3 +120,11 @@ const copyPlainObject = <T>(obj: Record<string, T>): Record<string, T> => {
   });
   return newObject;
 };
+
+export const copyProperties = <T>(source: T, target: T): void => {
```

Comment:
Few things here. Maybe better name would be `patch...` rather than `copy...`. When I see `copy` I would expect function to return me copied, not do any changes on the input data.

Also, nitpicking, but ordering matters too - `(target, source)` makes more sense because of `target <= source`. All resulting variables are always on the left side or first one.